ppdc/ppdc.cxx

```diff
@@ -16,437 +16,431 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 
-
 //
 // Local functions...
 //
 
-static void	usage(void) _CUPS_NORETURN;
-
+static void usage(void) _CUPS_NORETURN;
 
 //
 // 'main()' - Main entry for the PPD compiler.
 //
 
-int					// O - Exit status
-main(int  argc,				// I - Number of command-line arguments
-     char *argv[])			// I - Command-line arguments
+int                // O - Exit status
```

Comment:
The indentation is seriously messed up here. Please keep the variables aligned, using tabs, with the comments out at column 41 (5 tabs).

Also, for coding style we put a space between simple variables and unary operators (`i ++`, `ptr ++`) except when we are dereferencing a pointer (`*ptr++`). Indentation is *2* spaces, only 4 for code inside a `case` statement.